WcaOnRails/app/views/competitions/new.html.erb

```diff
@@ -1,14 +1,7 @@
 <% provide(:title, t('.create_competition')) %>
+<% provide(:include_gmaps, true) %>
```

Comment:
Does it work if we remove this line both from here and `edit.html` and move it to `_competition_form.html.erb` ?